### PR TITLE
Fixed deprecation warnings

### PR DESCRIPTION
**Pull request by Crecket**
updated dependencies to latest version and switched proptypes to separate lib to avoid deprecation warnings, added yarn lock file

### CODE REVIEW

**Comment by porfirioribeiro** [comment]
Please merge this PR as soon as possible @leandrowd , want to get rid of React warnings about prop-types!
I think you should add prop-types to dependencies

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-easy-swipe",
-  "version": "0.0.8",
+  "version": "0.0.9",
```
Comment (leandrowd) [inline package.json]: Can you return this to .8? I have a script that will bump it automatically.

File: package.json [hunk 2]
```diff
@@ -81,5 +81,8 @@
     "Mobile",
     "Swipe",
     "Touch"
-  ]
+  ],
+  "dependencies": {
+    "prop-types": "^15.5.8"
```
Comment (leandrowd) [inline package.json]: Please move it to devDependencies.

**Comment by leandrowd** [comment]
All done, published as 0.0.10. Thanks for the contribution @Crecket .

**Pull request merged**